mikael-andre/Graylog
# 1.0

# Features #

#### Verifications ####
- Check Internet connectivity
- Verify system compatibilities

#### OpenSSL ####
- Generate SSL private/public keys

#### YUM ####
- Create YUM repositories
- Create YUM cache

#### Network Time Protocol ####
- Install NTP service
- Add it on startup (Optional)

#### Core packages ####
- Install LSB (Linux Standard Base) packages
- Install network packages (tcpdump, scp, telnet, traceroute, etc...)

#### Secure SHell ####
- Configure OpenSSH
- Authenticate "root" user by using SSH keys

#### Postfix ####
- Configure it

#### Bourn Again SHell ####
- Configure it
- Configure /etc/hosts file

#### SELinux ####
- Disable it

#### Mongo ####
- Install and configure it
- Add it on startup (Optional)

#### JAVA ####
- Install JRE (Java Runtime Environment)

#### ElasticSearch ####
- Install and configure it
- Install HQ plugin to manage it (Optional)
- Add it on startup (Optional)

#### Graylog ####
- Server component
 - Install and configure it
 - Add it on startup (Optional)
- Web interface
 - Install and configure it
 - Add it on startup (Optional)

#### Nginx ####
- Install and configure it as a proxy
- Add it on startup (Optional)
